src/XoWithTestHelpers.js

```diff
@@ -13,6 +13,10 @@ export default class XoWithTestHelpers extends Xo {
     return userId;
   }
 
+  createUserWithoutStoringId(params) {
+    return super.call("user.create", params);
+  }
+
```

Comment:
IMHO these helpers are not necessary and introduce an abstraction which makes the tests more complex.

I think we should only use them when they provide a test-specific advantage such as automatically garbage collecting the users (for `createUser`).